tljh-voila-gallery/tljh_voila_gallery/templates/page.html

```diff
@@ -50,9 +50,9 @@ <h4 class="text-white">About</h4>
         <div class="col-sm-4 offset-md-1 py-4">
           <h4 class="text-white">Built with:</h4>
           <ul class="list-unstyled">
-            <li><a href="#" class="text-white">Voila</a></li>
-            <li><a href="#" class="text-white">IPywidgets</a></li>
-            <li><a href="#" class="text-white">The littlest JupyterHub</a></li>
+            <li><a href="https://github.com/QuantStack/voila" class="text-white">Voila</a></li>
+            <li><a href="https://github.com/jupyter-widgets/ipywidgets" class="text-white">ipywidgets</a></li>
+            <li><a href="https://github.com/jupyterhub/the-littlest-jupyterhub" class="text-white">The Littlest JupyterHub</a></li>
```

Comment:
This should point totljh.jupyter.org

Reply:
yeah, I guess they could all point to docs instead of GitHub repos